Test/Invoke-WhiskeyTask.Tests.ps1

```diff
@@ -3,6 +3,8 @@ Set-StrictMode -Version 'Latest'
 

```

Comment:
I don't understand why you made any changes to this test. You only changed one line in Invoke-WhiskeyTask that shouldn't have had any affect on tests. Can you give me some context?

Reply:
When I was making `TaskDefaults` a global config I added some new tests to this file at which point I also did some refactoring to make the tests cleaner. I ended up not needing the tests that I added, but left the refactoring. Should I revert these changes?

Reply:
Yes, please.